### ISSUE TITLE
When Tabby is running, new windows open behind older windows. This affects all applications in the OS (Linux Mint Cinnamon)

### ISSUE THREAD

**Issue by pacharanero** [Eugeny/tabby#9250]
**Describe the problem**:
* OS: Linux Mint 21.2 Cinnamon
* When Tabby is running, any new windows and dialogs will open *behind* existing windows.
* When Tabby is closed, the issue reverts immediately and new window behaviour is normal.

**To Reproduce**:
* Install Tabby via .deb
* Open Tabby
* Open browser (for an example window)
* Open a new browser window (eg `Ctrl + Shift + N` on Chromium-based browsers)
* This new window will open **behind** any existing windows but focus is lost from the existing (top) window.
* `Alt-Tab` shifts the focus back to the existing (top) window.
* A _further_ `Alt-Tab` is required to show the new window above the existing one.

None of this happens if Tabby is not running. It seems that somehow Tabby is reversing the display order of new windows.

This can be particularly problematic if an application opens a blocking modal _behind_ itself, as you can't see the new modal, you wouldn't know it was there, except that the top window will be unresponsive and it can appear as though the entire OS has frozen. Using `Alt-Tab`-ing a few times can sometimes reveal the rearmost window/modal/dialog which you can then interact with, unblocking the parent window again, and returning things to normal.

This is quite an annoying bug and although Tabby is by far the nicest terminal emulator I've used, it's starting to add inefficiency into daily workflows to the extent that Im considering other options. Thanks for Tabby though @Eugeny - I don't underestimate the work and effort that's gone into making such a great piece of open source software!

If I can supply any further information to help debug what is happening on Linux Mint then please do ask for what you need. I'm presuming this is only happening on Cinnamon, because if all users on all WMs/OSs were experiencing this I figure someone would have reported it already. I did search the Issues extensively before reporting.

**Comment by pacharanero** [comment]
I'm slightly surprised there is nobody else with a similar problem, there must be other Tabby users on Linux Mint?
Is there anything else I can do to help debug the issue? Could it be related to a different software package that I also have installed?

**Comment by NJNataren** [comment]
I am on Ubuntu 22.04.4 LTS, if I move to a new activities window (crtl+alt+right arrow), and right click on Tabby in the dash, the new Tabby instance opens up on the first workspace, which is quite annoying also if I want to be doing another task in the new workspace. I agree @pacharanero, tabby is a lovely terminal emulator in every other way, but this is hampering productivity. I am also surprised more people have not had these issues.

**Comment by denis3509** [comment]
I absolutely love Tabby, but I am experiencing the same issue. Tabby always opens new windows on the desktop where it was first launched, which makes me resort to using other terminals.

**Comment by j0hnm4r5** [comment]
I'm experiencing the same thing right now, with the added annoyance that all of the windows are behaving like a docked window. So not only are they hidden behind my intentional docked window, but even if I move them somewhere visible, they hide when they lose focus.

MacOS Tahoe 26.3
Tabby 1.0.230

EDIT:

Actually I'm pretty new to Tabby; are Docked windows not separate concepts from other windows? Are all windows actually supposed to hide if I use the `toggle-window` hotkey? That would be a surprising UX if so.
